### ISSUE TITLE
Reconcile Bulk DNA vs Bulk WES

### ISSUE THREAD

**Issue by elysekozlowski** [ncihtan/htan-portal#260]
The portal standards page and RFC describes Bulk DNA sequencing but the metadata templates are labeled Bulk WES. The standards page also mentions RNA sequencing

<img width="1265" alt="Screen Shot 2021-04-16 at 7 13 15 AM" src="https://user-images.githubusercontent.com/67977813/115037345-39a5be80-9e83-11eb-989a-8aa10ae40703.png">

**Comment by inodb** [comment]
Yeah this will need to be fixed in the schema - @milen-sage could you help with this one?

**Comment by milen-sage** [comment]
this is an artifact of the order in which the various data types were added... initially we were supporting WES, however based on RFC and other discussions it was easy to modify WES attributes to account for other DNA seq assays. So we started thinking of the WES component in the data model as bulk DNA seq, while the data model component remained named bulk WES. We can fix the schema for future uploads and update metadata templates. (note: this is lower priority relative to dbGaP)
